src/components/NodesTable.tsx

```diff
@@ -208,7 +213,28 @@ function MaticBalance({ address }: { address: string }) {
         }
     }, [address])
 
-    return balance || <Spinner color="blue" />
+    return (
+        (
+            <>
+                {balance}{' '}
+                {balance && minAmount && toBN(balance).isLessThan(toBN(minAmount)) && (
+                    <Tip
+                        handle={
+                            <TipIconWrap
+                                className="ml-1"
+                                $color="#ff5c00"
+                                $svgSize={{ width: '18px', height: '18px' }}
+                            >
+                                <JiraFailedBuildStatusIcon label="Error" />
+                            </TipIconWrap>
+                        }
+                    >
+                        Low MATIC
+                    </Tip>
+                )}
+            </>
+        ) || <Spinner color="blue" />
```

Comment:
You'll never see the spinner.

Reply:
OMG :D thanks!
![Facepalm](https://media.giphy.com/media/14aUO0Mf7dWDXW/giphy.gif)